SRA: runOnModule: bench_C
SRA: runOnModule: main
NUMBER OF UNKNOWN POINTERS: 2
34 Instrucoes
17 Ponteiros
1.952000e-03 Tempo
------------------------------------------------------------------------
/home/vitor/Ecosoc/llvm-3.5.0/lib/Ecosoc/llvm-sra/SAGE/sage/local/lib/python2.7/site-packages/sage/ext/interrupt/interrupt.so(+0x3c95)[0x7ffba1c50c95]
/home/vitor/Ecosoc/llvm-3.5.0/lib/Ecosoc/llvm-sra/SAGE/sage/local/lib/python2.7/site-packages/sage/ext/interrupt/interrupt.so(+0x3ce7)[0x7ffba1c50ce7]
/home/vitor/Ecosoc/llvm-3.5.0/lib/Ecosoc/llvm-sra/SAGE/sage/local/lib/python2.7/site-packages/sage/ext/interrupt/interrupt.so(+0x6279)[0x7ffba1c53279]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x10340)[0x7ffba5759340]
[0x3258f40]
------------------------------------------------------------------------
------------------------------------------------------------------------
Unhandled SIGSEGV: A segmentation fault occurred in Sage.
This probably occurred because a *compiled* component of Sage has a bug
in it and is not properly wrapped with sig_on(), sig_off().
Sage will now terminate.
------------------------------------------------------------------------
/home/vitor/Ecosoc/llvm-3.5.0/lib/Ecosoc/llvm-sra/SAGE/bin/sage: line 134: 24389 Segmentation fault      (core dumped) "$SAGE_ROOT/src/bin/sage" "$@"
